frontend/app/views/spree/checkout/_terms_and_conditions.en.html.erb

```diff
@@ -0,0 +1,64 @@
+____________________________________________________________
```

Comment:
I think text is appropriate. In this way we prevent inserting the html code (link, images ecc...) and the documents can be printed in well format.

ref https://github.com/solidusio/solidus/issues/1196#issuecomment-222467628
```
The terms and conditions should be in a format that can be printed or saved – therefore avoid pop-up windows and ensure that they fit within the width of the page and are presented in a way that they will print properly.
```